portfolio/src/main/webapp/script.js

```diff
@@ -62,3 +62,67 @@ function changeProfilePic() {
   document.getElementById('pfp-link').href = '/images/' + img;
   document.getElementById('pfp').src = '/images/' + img;
 }
+
+let xando = [
```

Comment:
Please add a comment explaining what this data structure represents.

Reply:
Done